mailpoet/lib/AdminPages/Pages/Upgrade.php

```diff
@@ -12,18 +13,33 @@ class Upgrade {
   /** @var WPFunctions */
   private $wp;
 
+  private CapabilitiesManager $capabilitiesManager;
+
   public function __construct(
     PageRenderer $pageRenderer,
-    WPFunctions $wp
+    WPFunctions $wp,
+    CapabilitiesManager $capabilitiesManager
   ) {
     $this->pageRenderer = $pageRenderer;
     $this->wp = $wp;
+    $this->capabilitiesManager = $capabilitiesManager;
   }
 
   public function render() {
     $data = [
       'current_wp_user' => $this->wp->wpGetCurrentUser()->to_array(),
     ];
+
+    // @todo change this after a/b test to keep only one of the pages
```

Comment:
Would be good to mention this in [MAILPOET-5986] or mention the ticket in the comment.

Reply:
Added to the ticket